src/EnergyPlus/Coils/CoilCoolingDXCurveFitSpeed.cc

```diff
@@ -371,7 +371,7 @@ void CoilCoolingDXCurveFitSpeed::size(EnergyPlus::EnergyPlusData &state)
     sizerCoolingCapacity.overrideSizingString(SizingString);
     if (this->original_input_specs.gross_rated_total_cooling_capacity_ratio_to_nominal < 1.0) {
         state.dataSize->DataDXCoolsLowSpeedsAutozize = true;
-        state.dataSize->DataConstantUsedForSizing = -1.0;
+        state.dataSize->DataConstantUsedForSizing = -999.0;
```

Comment:
I would think the proper reset for this would be the original declaration in DataSizing.hh. I am curious why this was changed.

    Real64 DataConstantUsedForSizing = 0.0;          // base value used for sizing inputs that are ratios of other inputs
    Real64 DataFractionUsedForSizing = 0.0;          // fractional value of base value used for sizing inputs that are ratios of other inputs

Reply:
BTW this is not a problem in design air flow rate sizing since there code [here](https://github.com/NREL/EnergyPlus/blob/develop/src/EnergyPlus/Autosizing/CoolingAirFlowSizing.cc#L72-L74) test is `>` not `>=`.

Reply:
The function `BaseSizer::initializeWithinEP()` resets them to 0.0, but there maybe sizing calculation that does not use this function. 

```
    this->dataConstantUsedForSizing = state.dataSize->DataConstantUsedForSizing;
    this->dataFractionUsedForSizing = state.dataSize->DataFractionUsedForSizing;
    state.dataSize->DataConstantUsedForSizing = 0.0; // reset here instead of in component model?
    state.dataSize->DataFractionUsedForSizing = 0.0;
```